### ISSUE TITLE
Quotes not formatted in component metadata

### ISSUE THREAD

**Issue by elpete** [jcberquist/commandbox-cfformat#6]
With the following settings file:
```json
{
    "strings.quote": "double"
}

```
Running the following file:
```cfc
component extends='coldbox.system.testing.BaseTestCase' appMapping='/' {

    function run() {
        describe( "about Suite", function() {
            beforeEach( function( currentSpec ) {
                setup();
            } );

            it( "index", function() {
                var event = execute( event = "about.index", renderResults = true );
                // expectations go here.
                expect( false ).toBeTrue();
            } );
        } );
    }

}
```
The quotes in the component metadata (`extends`, `appMapping`) are not converted.

**Comment by jcberquist** [comment]
Currently the string quote conversion is not done on strings used as tag attribute values (and the syntax treats component attributes as tag attributes). I could update the string formatting to do this, but I actually prefer (currently 😄 ) to double quote my tag attributes and single quote strings in cfscript. What would you think about adding another quote setting specifically for tag attributes?

**Comment by elpete** [comment]
This isn't prettier, so we don't need to be opinionated. 😉  I think a separate setting would work just fine.

**Comment by jcberquist** [comment]
I have published `v0.7.0` with the setting `strings.attributes.quote` (subject to change 😀 ) that should control tag attribute quotes.
